modules/nextflow/src/main/groovy/nextflow/fusion/FusionScriptLauncher.groovy

```diff
@@ -88,6 +88,8 @@ class FusionScriptLauncher extends BashWrapperBuilder {
             final work = toContainerMount(remoteWorkDir).toString()
             final result = new LinkedHashMap(10)
             result.FUSION_WORK = work
+            result.AZURE_BATCH_ACCOUNT_NAME=System.getenv("AZURE_BATCH_ACCOUNT_NAME")
+            result.AZURE_STORAGE_SAS_TOKEN=System.getenv("AZURE_STORAGE_SAS_TOKEN")
```

Comment:
Worth to retrieve those via [AzConfig](https://github.com/nextflow-io/nextflow/blob/e16c6778f6f4e57c56db5e8c933b098906bcdbf3/plugins/nf-azure/src/main/nextflow/cloud/azure/config/AzConfig.groovy#L71-L73)